### ISSUE TITLE
setValue does not work in Firefox 53.0

### ISSUE THREAD

**Issue by emilbader-ab** [webdriverio/webdriverio#1999]
## The problem

I'm unable to input text into an input field using `browser.setValue` in Firefox 53.0.
Works OK in Chrome 57 and Firefox 52.0.

## Environment

* WebdriverIO version: 4.6.2
* Selenium version: 3.3.1
* Geckodriver version: 0.15
* Node.js version: 6.6.0
* wdio testrunner
* running synchronous tests
* Additional wdio packages used (if applicable):
  * wido-dot-reporter: 0.0.6
  * wdio-junit-reporter: 0.2.0
  * wdio-mocha-framework: 0.5.9
  * wdio-screenshot: 0.5.2
  * wdio-selenium-standalone-service: 0.0.8
  * wdio-spec-reporter: 0.0.5
  * wdio-sync: 0.6.13

## WebdriverIO error message
```
Error: Expected [object Undefined] undefined to be a string
    at evalmachine.<anonymous>:1:-53
    at ContextifyScript.Script.runInThisContext (vm.js:25:33)
    at Object.exports.runInThisContext (vm.js:77:17)
    at node_modules/webdriverio/build/lib/commands/debug.js:54:43
    Build info: version: '3.3.1', revision: '5234b32', time: '2017-03-10 09:04:52 -0800'
    System info: host: '57221', ip: '10.12.12.175', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.4', java.version: '1.8.0_51'
    Driver info: org.openqa.selenium.firefox.FirefoxDriver
    Capabilities [{moz:profile=/var/folders/yh/qx48sn8j4gx168b_bwyk2fr9t2zmj3/T/rust_mozprofile.ZMTiKNwMs084, rotatable=false, timeouts={implicit=0, pageLoad=300000, script=30000}, pageLoadStrategy=normal, platform=ANY, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=false, browserVersion=53.0, platformVersion=16.5.0, moz:processID=93149, browserName=firefox, platformName=darwin}]
    Session ID: 85b9660c-d70e-bf45-a66c-a89a781bfb5a
    at elementIdValue("0", "hejehjehjkhkjsdhkjshfjksdhfkjsdhkf") - setValue.js:46:80
```
## Code To Reproduce Issue
```
describe('setValue test', () => {
            it('Google input value test', () => {
                browser.url('http://www.google.se/');
                browser.debug();
                // #lst-ib is the ID of the input field
                browser.setValue('#lst-ib', 'Hej'); // throws error
            });
        });
```

**Comment by TheMetalCode** [comment]
I see this too...although I'm going to guess the fix would occur in geckodriver (or a Firefox patch) and not so much in webdriverio.

**Comment by christian-bromann** [comment]
Sorry but this is a Geckodriver issue. Please report it there. Thanks!

**Comment by emilbader** [comment]
This has been fixed in Geckodriver 0.16.0 and depends on Selenium 3.4.0 or greater in order to work.
 
Read more about the details [here](https://github.com/mozilla/geckodriver/issues/659).
